client/src/Components/Pages/Dashboard/PersonalInfo/index.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import propTypes from 'prop-types';
+import emailImg from './images/email.svg';
+import phoneImg from './images/phone.svg';
+import docImg from './images/docs.png'
+import "./style.css";
+
+
+export default function index(props){
+const   {name ,city ,age ,favouritSubject,infoCompletion,email,mobile}= props;
+return(
+  <React.Fragment>
+  <div className="PersonalInfo">
+  <h2>Personal Info</h2>
+    <span className="info">Name: {name}</span>
+    <span className="info">Age:{age}</span>
+    <span className="info">City: {city}</span>
+    <span className="info">Favourit subject: {favouritSubject}</span>
+    <span className="info">Your Profile Complete: {infoCompletion}</span>
```

Comment:
If you're interested in how to render the white and green progress bar from the figma prototype, this is how I usually do it:
1) Create two divs, one nested inside the other.  Basically, an "outer" div (this is the white part) and an "inner" div (this is the green part, which shows the current progress)
2) With css, set the outer div to position: relative and inner div to position: absolute.  Give the outer div a width and height (387px wide, 38px high in figma), and give the inner div top: 0, left: 0, height: 100%.  Set their background-color and border-radius and all that :)
3) In the javascript, set the inner div's width style to be `infoCompletion`%.  Something like this:
```
<div className="progress-outer">
  <div className="progress-inner" style={{ width: `${ infoCompletion }%` }} />
</div>
```

Reply:
Great, That was very helpful.